be/src/util/mem_info.cpp

```diff
@@ -144,6 +146,8 @@ bool MemInfo::process_minor_gc() {
     // TODO add freed_mem
     SegmentLoader::instance()->prune();
 
+    freed_mem += tg_soft_memory_limit_gc(_s_process_minor_gc_size - freed_mem);
```

Comment:
Add more detailed comments for `process_minor_gc` and `process_full_gc`. GC order in each case.